src/python/pants/backend/rust/subsystems/rust.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+from __future__ import annotations
+
+import os
+
+from pants.option.option_types import StrListOption, StrOption
+from pants.option.subsystem import Subsystem
+from pants.util.ordered_set import OrderedSet
+
+
+class RustSubsystem(Subsystem):
+    options_scope = "rust"
+    help = "Options for Rust support."
+
+    toolchain = StrOption(
+        "--toolchain",
+        default="stable",
+        help=(
```

Comment:
For these multiple-implictly-concatenated string, use `strutil.py`'s `help_text`